include/sfx.h

```diff
@@ -55,9 +55,9 @@ typedef struct SfxBankEntry {
  * bank     1111000000000000    observed in audio code
  */
 
-#define DEFINE_SFX(enum, _1, _2, _3, _4) enum,
+#define DEFINE_SFX(_0, enum, _2, _3, _4, _5) enum,
 
-typedef enum SfxId {
+typedef enum {
```

Comment:
why?

Reply:
the enum change is recent, i imagine it was just copy pasted from an old repo that didnt have it yet